### PR TITLE
Add Mikrotik & Zyxel device types

### PR DESCRIPTION
**Pull request by zupy666**
Add device types:
MikroTik:
- CRS109-8G-1S-2HnD-IN
- RB951Ui-2nD
Zyxel:
- PMG1005-T20B

### CODE REVIEW

File: device-types/MikroTik/CRS109-8G-1S-2HnD-IN.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: MikroTik
+model: CRS109-8G-1S-2HnD-IN
+slug: crs109-8g-1s-2hnd-in
```
Comment (danner26) [inline device-types/MikroTik/CRS109-8G-1S-2HnD-IN.yaml:4]: Please append `mikrotik-` to the slug

File: device-types/MikroTik/RB951Ui-2nD,yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: MikroTik
+model: RB951Ui-2nD-tst
+slug: rb951ui-2nd-tst
```
Comment (danner26) [inline device-types/MikroTik/RB951Ui-2nD,yaml:4]: Please append `mikrotik-` to the slug
The full slug for this should be `mikrotik-hap`

File: device-types/MikroTik/RB951Ui-2nD,yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: MikroTik
+model: RB951Ui-2nD-tst
```
Comment (danner26) [inline device-types/MikroTik/RB951Ui-2nD,yaml:3]: The model name should be `hAP`

File: device-types/Zyxel/PMG1005-T20B.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Zyxel
+model: PMG1005-T20B
+slug: zyxel-pmg1005-t20b
+part_number: 'SKU : PMG1005-T20C'
```
Comment (danner26) [inline device-types/Zyxel/PMG1005-T20B.yaml:5]: It looks like the T20B and T20C are different models, is that true? If so they should be different device types

Also, please make sure the part number does not have `SKU : ` in it. It should just be the applicable part number

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Please review the requested changes, also please make sure all tests succeed. I am going to mark this PR as a draft, and once you are ready and all tests pass please mark it ready for review/ping me to re-review

File: device-types/MikroTik/CRS109-8G-1S-2HnD-IN.yaml (hunk 1 above)
Comment (zupy666) [reply device-types/MikroTik/CRS109-8G-1S-2HnD-IN.yaml]: done

File: device-types/MikroTik/RB951Ui-2nD,yaml (hunk 2 above)
Comment (zupy666) [reply device-types/MikroTik/RB951Ui-2nD,yaml]: done

File: device-types/MikroTik/RB951Ui-2nD,yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: MikroTik
+model: RB951Ui-2nD-hAP
```
Comment (danner26) [inline device-types/MikroTik/RB951Ui-2nD,yaml]: The model should just be `hAP`

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/MikroTik/RB951Ui-2nD,yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: MikroTik
+model: MikroTik-hAP
```
Comment (danner26) [inline device-types/MikroTik/RB951Ui-2nD,yaml:3]: Please remove `MikroTik-` from the model name, this only should be present in the slug

File: device-types/Zyxel/PMG1005-T20B.yaml (hunk 4 above)
Comment (zupy666) [reply device-types/Zyxel/PMG1005-T20B.yaml]: done

File: device-types/MikroTik/RB951Ui-2nD,yaml (hunk 3 above)
Comment (zupy666) [reply device-types/MikroTik/RB951Ui-2nD,yaml]: done

File: device-types/MikroTik/RB951Ui-2nD,yaml (hunk 6 above)
Comment (zupy666) [reply device-types/MikroTik/RB951Ui-2nD,yaml]: done

**Review by danner26 — APPROVED** [review]

**Pull request merged**